docs/sphinx/sections/api/apidocs/libraries/dagster-pipes.rst

```diff
@@ -1,51 +1,94 @@
 Pipes (dagster-pipes)
------------------------
-
-This library is intended for inclusion in an external process that integrates
-with Dagster using the Pipes protocol.
+=====================
 
 .. currentmodule:: dagster_pipes
 
+The ``dagster-pipes`` library is intended for inclusion in an external process that integrates with Dagster using the `Pipes </concepts/dagster-pipes>`_ protocol. This could be in an environment like Databricks, Kubernetes, or Docker. Using this library, you can write code in the external process that streams metadata back to Dagster.
```

Comment:
I think this can also mention that pipes passes parameters _to_ the external process from the orchestration environment and you can access them there. It's a key value prop of the system.